kotekan/kotekan.cpp

```diff
@@ -453,8 +453,9 @@ int main(int argc, char** argv) {
     });
 
     rest_server.register_get_callback("/kill", [&](connectionInstance& conn) {
-        INFO("/kill endpoint called, raising SIGINT to shutdown the kotekan system process.");
-        raise(SIGINT);
+        ERROR("/kill endpoint called, raising SIGINT to shutdown the kotekan system process.");
+        set_error_message("/kill endpoint called, raising SIGINT to shutdown the kotekan system process.");
```

Comment:
I'd remove the SIGINT bit just to make the final message more to the point since this is a special (but common) case. 
```suggestion
        set_error_message("/kill endpoint called.");
```